tests/tests/test_unit/test_tools/test_system.py

```diff
@@ -14,12 +14,12 @@
 
 def test_get_subdirectories():
     subdirs = system.get_subdirectories(data_dir)
-    assert len(subdirs) == 9
+    assert len(subdirs) == 10
```

Comment:
Why is this now 10 and not 9? I can't see why this would have changed but I'm also not super knowledgeable about what `cellfinder` is meant to be doing here.

Reply:
This test checks the number of subdirectories in `tests/data` and I've added a new `analyse` subdirectory as part of the regression test introduced in the PR.